packages/manager/modules/veeam-enterprise/src/dashboard/license/license.controller.js

```diff
@@ -10,11 +10,19 @@ export default class VeeamEnterpriseLicenseCtrl {
   }
 
   dismissModal() {
+    this.trackClick(
+      `license::${this.action === 'register' ? 'activate' : 'update'}::cancel`,
```

Comment:
Maybe we could avoid code repetition (L14 & L22/L23)
```suggestion
      this.getLicenseTag('cancel'),
```

Reply:
It's apply